pyproject.toml

```
@@ -1,9 +1,9 @@
 [tool.black]
-extend_exclude = '_pb2(_grpc)?\.(py|pyi)$|ni_measurementlink_generator/|examples/'
+extend_exclude = '.tox/,_pb2(_grpc)?\.(py|pyi)$|ni_measurementlink_generator/|examples/'
```

Black and ni-python-styleguide use different formats here. Black wants a single regex, and ni-python-styleguide wants a comma-separated list of fnmatch patterns. 😩

```suggestion
extend_exclude = '\.tox/|_pb2(_grpc)?\.(py|pyi)$|ni_measurementlink_generator/|examples/'
```

---

Corrected.